docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,24 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/material-ui/react-modal/) and [`Popper`](/material-ui/react-popper/) components.
+`Portal` is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
+It gives you the functionality of `createPortal()` in a convenient component form.
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+> 💡 According to [the React documentation](https://reactjs.org/docs/portals.html), portals are useful when "you need the child element to visually 'break out' of its container."
 
-## Example
+The `Portal` component is used internally by Material UI's [`Modal`](/components/modal/) and [`Popper`](/components/popper/) components.
 
-{{"demo": "SimplePortal.js"}}
+## Basic usage
+
+Normally, child elements are appended to the nearest parent node.
+But sometimes it's necessary to mount a child to a different location in the DOM.
+
+The `Portal` component accepts a `container` prop that passes a `ref` to the DOM node where its children will be mounted.
```

Comment:
I hope I explained this correctly! It's a lot of jargon to pin down. 😅

Reply:
That could just be: 
```suggestion
Portal accepts a `container` prop that passes a `ref` to the DOM node where its children will be mounted.
```
(We've already established that Portal is a React component.)

Also is that the right "direction"? Isn't it the ref to the dom node where the children will be mounted passed to the Portal with this prop?